translate/rus.json

```diff
@@ -285,8 +285,8 @@
 "play as":"играть как",
 "play every":"играть каждый",
 "player":"игрок",
-"Please consider whitelisting [TCEC_URL] in your ad blocker --- or send a donation.":"Пожалуйста, рассмотрите белый список [TCEC_URL] в вашем блокировщике рекламы --- или отправьте пожертвование.",
-"points + direct encounters":"очки + прямые встречи",
+"Please consider whitelisting [TCEC_URL] in your ad blocker --- or send a donation.":"Пожалуйста, добавьте [TCEC_URL] в белый список  в вашем блокировщике рекламы --- или отправьте пожертвование.",
+"points + direct encounters":"очки + личные встречи",
```

Comment:
Line 288: I don't know why @Aloril refused to change this line. Previous translation said to me 'look through your (URL) whitelist in your adblocker or send a donation.' New one says 'add (URL) to the whitelist of your adblocker or send a donation'.
Line 289: This change was last-second change by @Aloril. Please show where this this phrase is used in GUI, I will do a proper correction of the translation.